ccloud/Dockerfile-local

```diff
@@ -17,7 +17,7 @@ ARG CP_VERSION
 
 FROM confluentinc/cp-server-connect-base:$CP_VERSION
 
-ARG CONNECTOR_VERSION=5.5.0-SNAPSHOT
+ARG CONNECTOR_VERSION=6.0.0-SNAPSHOT
```

Comment:
@rspurgeon should this be `6.0.0` instead?

Reply:
@ybyzek .. yes.  Thank you! 🤦

Reply:
@ybyzek did you see my correction for this ?